test/uk/gov/hmrc/mtdsatestsupportapi/controllers/requestParsers/DeleteStatefulTestDataRequestParserSpec.scala

```diff
@@ -25,16 +26,17 @@ import uk.gov.hmrc.mtdsatestsupportapi.models.request.deleteStatefulTestData.{De
 class DeleteStatefulTestDataRequestParserSpec extends UnitSpec {
 
   val vendorClientId = "some_id"
+  val nino = "TC663795B"
   val requestBody: JsObject = Json.obj("exampleBody" -> "someValue")
   implicit val correlationId: String = "X-123"
 
   "DeleteStatefulTestDataRequestParser" should {
-    val data = DeleteStatefulTestDataRawData(vendorClientId, Some(requestBody))
+    val data = DeleteStatefulTestDataRawData(vendorClientId, Some(nino))
     "return a request object" when {
       "valid request data is supplied" in new Test {
         MockDeleteStatefulTestDataValidator.validate(data).returns(Nil)
 
-        parser.parseRequest(data) shouldBe Right(DeleteStatefulTestDataRequest(vendorClientId, Some(requestBody)))
+        parser.parseRequest(data) shouldBe Right(DeleteStatefulTestDataRequest(vendorClientId, Some(Nino(nino))))
```

Comment:
Probably ought to check the happy path for both with and without nino cases (because the implementation itself -- and not a mocked dependency) maps over the nino Option.